doc/man7/proxy-certificates.pod

```diff
@@ -116,7 +116,7 @@ two commands:
 
     openssl x509 -req -CAcreateserial -in proxy.req -out proxy.crt \
         -CA user.crt -CAkey user.key -days 7 \
-        -extfile proxy.cnf -extensions v3_proxy1
+        -extfile proxy.cnf -extensions proxy
```

Comment:
Please note that above in the config file snippet example v3_proxy is used as the name of the extension section. So perhaps I would drop this change here, or it should be changed above as well.

Reply:
Fixed the CN's to match; thanks!

Reply:
You've misunderstood me - I was talking about the configuration file snippet in the "Creating proxy certificates" section in this manual page.

Reply:
Fixed the config file example too, now.